isolator.py

```diff
@@ -102,15 +102,15 @@ def filter(self, record):
             return (record.levelno == self.passlevel)
 
 
-# h1 = logging.StreamHandler(sys.stdout)
-# f1 = SingleLevelFilter(logging.INFO, False)
-# # h1.addFilter(f1)
-# root_logger = logging.getLogger()
-# root_logger.addHandler(h1)
-# h2 = logging.StreamHandler(sys.stderr)
-# f2 = SingleLevelFilter(logging.INFO, True)
-# # h2.addFilter(f2)
-# root_logger.addHandler(h2)
+h1 = logging.StreamHandler(sys.stdout)
+f1 = SingleLevelFilter(logging.INFO, False)
+# h1.addFilter(f1)
+root_logger = logging.getLogger()
+root_logger.addHandler(h1)
+h2 = logging.StreamHandler(sys.stderr)
+f2 = SingleLevelFilter(logging.INFO, True)
+# h2.addFilter(f2)
+root_logger.addHandler(h2)
```

Comment:
This undoes the disabling of this by Matthias in ([71f2813](https://github.com/illinois-ceesd/drivers_y2-isolator/pull/6/commits/71f2813097b059c8858a5f3311699a0ca98d786a).  I suspect he needs this block disabled to effectively debug.